app/models/ec_numbers.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: ec_numbers
+#
+#  ec_number 		:string(12)       not null, primary key
+#  name	            :string(160)      not null
```

Comment:
Did you use the annotate gem to generate these comments? The spacing is a bit messed up.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3369) by @bmesuere on Mon Oct 19 2015 at 14:47._